web-app/src/i18n/en-us/tournament.ts

```diff
@@ -0,0 +1,15 @@
+export default {
+  name: 'Tournament Name',
+  team: 'Team',
+  teams: 'Teams List',
```

Comment:
```suggestion
  teamList: 'Team List',
```
teams != "team list"
When you will want to use `teams` word there may be some problems